### DISCUSSION TITLE
Groups

### DISCUSSION THREAD

**Discussion by btqsstudios** [keiwando/toonsquid#365] [Ideas]
I think you should be able to make groups in the library section, that way it’s way easier to get what your looking for, also I think they should allow you to put folders in folders  and of course somthing pretty obvious is to be able to name your folders.

these will help with a lot of stuff in different context like
You and someone else animate on the some iPad and you two have different are styles, and It could be as simple as you have two different art styles yourself.

**Comment by ota-works** [comment]
This would be great for organization! Great suggestion, dude!!!
